crm_deduplicate_acl/wizards/partner_merge_view.xml

```diff
@@ -3,29 +3,30 @@
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
 
 <openerp>
-<data>
+    <data>
 
-<record id="base_partner_merge_automatic_wizard_form" model="ir.ui.view">
-    <field name="name">Restrict automatic merge access</field>
-    <field name="model">base.partner.merge.automatic.wizard</field>
-    <field
-        name="inherit_id"
-        ref="crm.base_partner_merge_automatic_wizard_form"/>
-    <field name="arch" type="xml">
-        <xpath expr="//button[@name='automatic_process_cb']"
-               position="attributes">
-            <attribute name="groups">
-                crm_deduplicate_acl.group_automatically
-            </attribute>
-        </xpath>
-        <xpath expr="//button[@name='update_all_process_cb']"
-               position="attributes">
-            <attribute name="groups">
-                crm_deduplicate_acl.group_automatically
-            </attribute>
-        </xpath>
-    </field>
-</record>
+        <record id="base_partner_merge_automatic_wizard_form" model="ir.ui.view">
```

Comment:
OK for not adding line breaks on the attribute itself, but please don't indent all of them.